server/okta.ts

```diff
@@ -0,0 +1,56 @@
+import { joinUrl } from '@guardian/libs';
+import { z } from 'zod';
+import { log } from './log';
+import type { OktaValidationResponse } from './middleware/oktaServerSideAuthMiddleware';
+import type { OktaConfig } from './oktaConfig';
+
+const OktaUserInfo = z.object({
+	legacy_identity_id: z.optional(z.string()),
+	email: z.optional(z.string()),
+	name: z.optional(z.string()),
+});
```

Comment:
same problem here: this zod schema would pass for any given object/json passed into this as all the types are optional

also this is the same as IdTokenClaims